src/main/java/banking/Main.java

```diff
@@ -0,0 +1,19 @@
+package banking;
+
+import banking.configuration.DBManager;
+import banking.dao.AccountDAO;
+
+public class Main {
+    public static void main(String[] args) {
+        if (args.length == 2 && args[0].equals("-fileName")) {
+            DBManager connector = new DBManager(args[1]);
+            AccountDAO accountDAO = new AccountDAO(connector);
+            BankingSystem bankingSystem = new BankingSystem(accountDAO);
+            bankingSystem.showMenu();
+        } else {
+            throw new IllegalArgumentException("""
```

Comment:
Can you use just a regular String here instead of the text block?

Reply:
fixed